feat: move envrc to top level

smartcontracts opened the pull request.

smartcontracts committed:
feat: move envrc to top level

tynes commented on .envrc.example:
```diff
@@ -0,0 +1,56 @@
+export DEPLOYMENT_CONTEXT=getting-started
+
+# Optional Tenderly details for simulation link during deployment
+export TENDERLY_PROJECT=
```
I don't believe these are used anymore, worth a quick `grep`

tynes commented on .gitignore:
```diff
@@ -30,6 +30,7 @@ packages/contracts-bedrock/deployments/anvil
 .env
 .env*
 !.env.example
```
Do we still have a `.env.example`?

smartcontracts committed:
feat: add getting started config script

Adds a new script to contracts-bedrock that generates the deploy
config for the getting started network. Removes the getting started
config from the repository and avoids unnecessary work for the user
while following the guide.

tynes commented on packages/contracts-bedrock/scripts/getting-started-config.sh:
```diff
@@ -0,0 +1,97 @@
+
+  "l1StartingBlockTag": "$blockhash",
+
+  "l1ChainID": 5,
```
still doing goerli?

smartcontracts committed:
feat: add getting started wallet script

Adds a script to contracts-bedrock that generates the four wallets
required for the getting started guide.

smartcontracts committed:
fix: add config files to op-node gitignore

Adds the common config files to op-node gitignore since these get
created frequently and shouldn't be committed.

smartcontracts committed:
fix: strip 0x prefix when loading pk in op-node

smartcontracts committed:
fix: simplify loading environment vars

smartcontracts committed:
fix: remove private key override

trianglesphere commented on op-node/p2p/cli/load_signer.go:
```diff
@@ -19,7 +20,7 @@ func LoadSignerSetup(ctx *cli.Context) (p2p.SignerSetup, error) {
 		// Mnemonics are bad because they leak *all* keys when they leak.
 		// Unencrypted keys from file are bad because they are easy to leak (and we are not checking file permissions).
-		priv, err := crypto.HexToECDSA(key)
+		priv, err := crypto.HexToECDSA(strings.TrimPrefix(key, "0x"))
```
Nice fix, but seems unrelated to this PR

smartcontracts committed:
feat: move scripts to getting-started folder
